### PR TITLE
Add support for nginx order group

### PR DESCRIPTION
**Pull request by robdimsdale**
Signed-off-by: Sophie Wigmore <swigmore@vmware.com>

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by robdimsdale** [comment]
This is blocked on a release of [`php-nginx`](github.com/paketo-buildpacks/php-nginx) being available - the integration tests will fail until then.

File: build.go [hunk 1]
```diff
@@ -46,26 +46,43 @@ func Build(procs ProcMgr, logger scribe.Emitter) packit.BuildFunc {
 		}
 		layer.Launch = true
 
+		httpdConfPath, httpdOk := os.LookupEnv("PHP_HTTPD_PATH")
```
Comment (robdimsdale) [inline build.go:49]: @sophiewigmore I think we can simplify this by changing `os.LookupEnv` to `os.Getenv`, as we treat the empty and unset conditions equally. We can keep the existing tests (which test both empty and unset) but simplify the code.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
-A Cloud Native Buildpack for running HTTPD, Nginx, and/or FPM start commands
-for PHP apps.
+A Cloud Native Buildpack for running HTTPD, Nginx, and FPM start commands for
+PHP apps.
 
 ## Behavior
 
 This buildpack will always participate if it's `requirements` are met. In the
 HTTPD server case `requires` `php`, `php-fpm` optionally, `httpd`, and
 `httpd-config`. In the Nginx case, it will require `nginx` and `nginx-config`
```
Comment (thitch97) [inline README.md:11]:
```suggestion
`php-httpd-config`. In the Nginx case, it will require `nginx` and `php-nginx-config`
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
-instead of `httpd` and `httpd-config`. These requirements will be met when used
-in conjunction with the other buildpacks in the Paketo PHP language family.
+instead of `httpd` and `httpd-config`.
```
Comment (thitch97) [inline README.md:12]:
```suggestion
instead of `httpd` and `php-httpd-config`.
```

File: integration/httpd_test.go [hunk 4]
```diff
@@ -98,18 +104,7 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
...
-					return cLogs.String()
-				}).Should(
-					And(
-						MatchRegexp(`Apache\/\d+\.\d+\.\d+ \(Unix\) configured -- resuming normal operations`),
-						MatchRegexp(`fpm is running, pid \d+`),
-						ContainSubstring(`ready to handle connections`),
-					),
-				)
-				Eventually(container).Should(Serve(ContainSubstring("Hello World!")).OnPort(8080))
+				Eventually(container).Should(Serve(ContainSubstring("SUCCESS: date loads.")).OnPort(8080).WithEndpoint("/index.php?date"))
```
Comment (thitch97) [inline integration/httpd_test.go:107]: Just to confirm, is this change predicated on the idea that if the endpoint is hit successfully fpm must also be working correctly?

Comment (robdimsdale) [reply integration/httpd_test.go:107]: yeah, you got it. Without fpm running the php wouldn't render successfully.

**Review by thitch97 — APPROVED** [review]

**Pull request merged**